website/src/components/Edit/InputForm.spec.tsx

```diff
@@ -0,0 +1,24 @@
+import { describe, expect, test } from 'vitest';
+
+import { EditableSequences } from './InputForm';
+
+describe('InputForm', () => {
+    test('Empty editable sequences produces `undefined`', () => {
+        const emptyEditableSequences = EditableSequences.empty();
+        expect(emptyEditableSequences.getSequenceFasta('subId')).toBeUndefined();
```

Comment:
Maybe outside the scope of this PR but I think it would be good to rename the `getSequenceFasta` function - in my understanding it produces fasta objects with the submissionId it is given - so `createFastaWithId` would be a better name.

Also to verify my understanding each editableSequences object only contain the contents of one sample? because this function will give all sequences in the editable sequence object the same submissionId - which I don't think we want? It might be good to rename or document that as well for maintainability.